Makefile

```diff
@@ -24,6 +24,13 @@ export TF_IN_AUTOMATION="true"
 export TF_VAR_MATURITY=${MATURITY}
 export TF_VAR_DEPLOY_NAME=${DEPLOY_NAME}
 PYTHON_VER ?= python3
+
+
+CIRRUS_DAAC_BRANCH := $(shell git -C $(DAAC_DIR) rev-parse --abbrev-ref HEAD)
+CIRRUS_CORE_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+CIRRUS_DAAC_TAG := $(shell git -C $(DAAC_DIR) describe --tags --abbrev=0)
+CIRRUS_CORE_TAG := $(shell git describe --tags --abbrev=0)
+
```

Comment:
I think the branch name actually isn't what we want because the actual code pointed to by a branch can easily change. I think commit hash would be more appropriate if the commit being deployed doesn't have a tag. Here's something that might theoretically give us what we want (TEST IT FIRST BEFORE YOU COPY/PASTE!)

Get tags pointing at this commit. If there are multiple, just take one:
```
git tag --points-at HEAD | head -n1
```

Get the short version of the commit hash:
```
git rev-parse --short HEAD
```

Combine it in makefile with an `$(or)`:
```
CIRRUS_CORE_VERSION := $(or $(shell git tag --points-at HEAD | head -n1),$(shell git rev-parse --short HEAD))
```